client/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
Does this file need to exist?

Reply:
Good question... that file was generated by react-scripts when I updated to typescript

Reply:
According to [this article](https://www.sitepoint.com/react-with-typescript-best-practices/):
"The react-app-env.d.ts references the types of react-scripts, and helps with things like allowing for SVG imports."

I'll leave it in case it's important